### PR TITLE
lint: add `no-extra-parens`

### PR DESCRIPTION
**Pull request by trim21**
dprint will format these in new version, so remove them with eslint

https://eslint.org/docs/latest/rules/no-extra-parens

we can remove this after dprint is updated

### CODE REVIEW

**Comment by quisquous** [comment]
Maybe `lint` and not `style`?

File: ui/fisher/fisher-ui.js [hunk 1]
```diff
@@ -75,7 +75,7 @@ export default class FisherUI {
       bar.centertext = row.getAttribute('data-fish');
 
       // Step one: fill until the minimum time
-      if ((min && min !== 'undefined') && (max && max !== 'undefined')) {
+      if (min && min !== 'undefined' && (max && max !== 'undefined')) {
```
Comment (quisquous) [inline ui/fisher/fisher-ui.js:78]:
```suggestion
      if (min && min !== 'undefined' && max && max !== 'undefined') {
```

I love that it only caught one of the parens here??

**Comment by trim21** [comment]
This PR just change eslintrc and run `lintfix`, no other manually changes.

**Comment by trim21** [comment]
and this also avoid https://github.com/quisquous/cactbot/pull/4954#discussion_r1002784695

**Pull request merged**